ramls/customField.json

```diff
@@ -28,6 +34,39 @@
       "type": "string",
       "description": "The description of the custom field",
       "example": "Provide a department"
+    },
+    "maxSize": {
+      "type": "integer",
+      "description": "The text field maximum length",
+      "example": 42
+    },
+    "default": {
+      "type": "boolean",
+      "description": "Defines if the custom field is default",
+      "example": false
+    },
+    "defaults": {
```

Comment:
Can we rename this to "defaultValues"? Thanks!

Reply:
The schema for the custom fields will be changed(there is no issue created yet) and this field will be included in another object and renamed in the scope of the issue.